### ISSUE TITLE
Use markdown files in SvelteKit

### ISSUE THREAD

**Issue by janosh** [pngwn/MDsveX#389]
I'd like to deploy markdown documentation with SvelteKit where `src` in `<img src="/img.svg" alt=Image" />` needs to be an absolute link that assumes the asset directory (default: `'./static'`) as root.

For the markdown files to display assets correctly on GitHub, the `src` needs to be `![Image](static/img.svg)` instead. Is there a rehype plugin or something that can change the `src` from `'static/img.svg'` to `'/img.svg'` on the fly?

**Comment by akauppi** [comment]
I don't see why your question needs to be in the MDsveX issues - it reads more like a StackOverflow entry.

**Comment by janosh** [comment]
@akauppi Fair point. The easiest solution is probably to use raw.githubusercontent.com instead local files.

**Comment by pngwn** [comment]
I think this is a documentation issue that I'd like to resolve when I have a good solution. 

We may migrate the kit/ svelte docs to mdsvex soon so that could be a good test bed for some of these problems. 

Going to reopen to track.
